plugin/src/functionalTest/kotlin/com/automattic/android/applystyle/ApplyStylePluginFunctionalTest.kt

```diff
@@ -9,17 +9,17 @@ import kotlin.test.Test
 import kotlin.test.assertTrue
 
 /**
- * A simple functional test for the 'com.automattic.android.applystyle.greeting' plugin.
+ * A simple functional test for the plugin.
  */
-class ApplystylePluginFunctionalTest {
+class ApplyStylePluginFunctionalTest {
```

Comment:
There's a problem with the functional tests! I didn't had the time to elaborate much on it (there's a comment that you have found  https://github.com/wordpress-mobile/apply-style-gradle-plugin/pull/1/files#r666683180)

What happens is that during the functional tests I can't use any of the code of the plugin because there's a runtime crash with gradle complaining that I couldn't find the detekt code.

I tried adding the dependency to the functional project but I didn't find the way to do it yet. This is why the functional tests can't use much of the plugin at this point.